docs/style.json

```diff
@@ -8,7 +8,7 @@
     },
     "mapterhorn-terrain": {
       "type": "raster-dem",
-      "url": "https://tunnel.optgeo.org/martin/mapterhorn",
+      "url": "pmtiles://https://download.mapterhorn.com/",
```

Comment:
this should be "url": "pmtiles://https://download.mapterhorn.com/planet.pmtiles",